### DISCUSSION TITLE
Does detail router support spacing between different cut layers?

### DISCUSSION THREAD

**Discussion by davidcarails** [The-OpenROAD-Project/OpenROAD#1870] [General]
I have seen an example of a via/cut spacing rule between different layers.
For example : v4 space v5  >= 38
Can/does the detailed router support this?

**Comment by maliberty** [comment]
Its helpful to ask about a specific LEF rule but yes it does support cut spacing between layers.

**Reply by davidcarails** [reply]
Can you provide an example of the syntax?

**Comment by maliberty** [comment]
![image](https://user-images.githubusercontent.com/761514/169117659-453ea3a8-10cb-4794-bb41-16558182e3f8.png)

**Reply by davidcarails** [reply]
Thank you.
